lib/preview/previewUtils.js

```diff
@@ -39,9 +39,38 @@ const stringifyQuery = (query) => {
     .map(key => `${key}=${query[key]}`)
     .join('&');
 }
+
```

Comment:
Currently we return the page straight out from the render, so I just splice the script right into the end of the body. If we end up changing how we do this - I think an iframe might make sense - some of the logic here for splicing the script in will have to change with it. But the underlying mechanic stays the same.